container/go/cmd/puller/puller.go

```diff
@@ -100,23 +107,31 @@ func pull(imgName, dstPath, format, cachePath string, platform v1.Platform) erro
 	}
 
 	// Image file to write to disk, either a tarball, OCI layout, or both.
+	ociPath := path.Join(dstPath, ociImageDir)
+	tarPath := path.Join(dstPath, imageDir)
```

Comment:
I would call this legacyPath as it stores the legacy image as well as the tarball.